--- docker/env_rpm_optimize_image.conf.old	2024-02-07 18:11:22.996363162 +0700
+++ docker/env_rpm_optimize_image.conf	2024-02-07 18:04:24.558333612 +0700
@@ -16,7 +16,7 @@
 PASSWORD_PGS_CONTAINER_REGISTRY=
 
 PGSPIDER_BASE_POSTGRESQL_VERSION=16
-PGSPIDER_RELEASE_VERSION=4.0.0
+PGSPIDER_RELEASE_VERSION=16.0
 RPM_DISTRIBUTION_TYPE="rhel8"
 
 ACCESS_TOKEN=
--- docker/create_rpm_binary.sh.old	2024-02-07 18:12:46.810369081 +0700
+++ docker/create_rpm_binary.sh	2024-02-07 17:59:04.791311030 +0700
@@ -9,7 +9,12 @@
     wget https://www.postgresql.org/files/documentation/pdf/16/postgresql-16-A4.pdf
 fi
 
-# Create Docker image for creating RPM file of PGSpider.
+# download postgres
+if [[ ! -f postgresql-16.0.tar.bz2 ]]; then
+    wget https://ftp.postgresql.org/pub/source/v16.0/postgresql-16.0.tar.bz2
+fi
+
+# Create Docker image for creating RPM file of Postgres.
 docker build -t $IMAGE_NAME_RPM \
         --build-arg proxy=${proxy} \
         --build-arg no_proxy=${no_proxy} \
@@ -25,7 +30,7 @@
                 -u "$(id -u $USER):$(id -g $USER)" \
                 -e LOCAL_UID=$(id -u $USER) \
                 -e LOCAL_GID=$(id -g $USER) $IMAGE_NAME_RPM /bin/sh \
-                -c "cp /home/user1/rpmbuild/RPMS/x86_64/pgspider16*.rpm /tmp/"
+                -c "/root/rpmbuild/RPMS/x86_64/postgresql16*.rpm /tmp/"
 rm -f $RPM_ARTIFACT_DIR/*-debuginfo-*.rpm
 
 # # Push rpm binary to registry
--- docker/Dockerfile_rpm.old	2024-02-07 18:12:31.617368008 +0700
+++ docker/Dockerfile_rpm	2024-02-07 17:38:56.093225672 +0700
@@ -35,18 +35,11 @@
 RUN echo "user1:user1" | chpasswd
 RUN echo "user1 ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
 
-# Transfer files: PGSpider source code
-RUN mkdir -p /home/user1/pgspider-${PGSPIDER_RELEASE_VERSION}
-COPY ./ /home/user1/pgspider-${PGSPIDER_RELEASE_VERSION}
-
-RUN cp /home/user1/pgspider-${PGSPIDER_RELEASE_VERSION}/rpm/* /home/user1
-RUN cp /home/user1/pgspider-${PGSPIDER_RELEASE_VERSION}/postgresql-16-A4.pdf /home/user1
-
-RUN rm -rf  /home/user1/pgspider-${PGSPIDER_RELEASE_VERSION}/.git                  \
-            /home/user1/pgspider-${PGSPIDER_RELEASE_VERSION}/rpm                   \
-            /home/user1/pgspider-${PGSPIDER_RELEASE_VERSION}/postgresql-16-A4.pdf  \
-            /home/user1/pgspider-${PGSPIDER_RELEASE_VERSION}/PGSpider_rpm          \
-            /home/user1/pgspider-${PGSPIDER_RELEASE_VERSION}/centos-gpg-keys-8-3.el8.noarch.rpm
+# Transfer files for postgres
+WORKDIR /home/user1
+COPY ./rpm/* /home/user1
+COPY ./postgresql-16.0.tar.bz2 /home/user1
+COPY ./postgresql-16-A4.pdf /home/user1
 
 # Change permission
 RUN chown user1:user1 -R /home/user1/pgspider-${PGSPIDER_RELEASE_VERSION}
--- rpm/PGSpider.spec.old	2024-02-07 18:13:01.304370104 +0700
+++ rpm/PGSpider.spec	2024-02-07 17:52:45.241284226 +0700
@@ -5,7 +5,6 @@
 %global pgpackageversion 16
 %global prevmajorversion 15
 %global sname postgresql
-%global sname_pgs pgspider
 %global pgbaseinstdir	/usr/pgsql-%{pgmajorversion}
 
 %global beta 0
@@ -61,20 +60,20 @@
 %global __requires_exclude ^perl\\((PostgresVersion|PostgresNode|RecursiveCopy|SimpleTee|TestLib|PostgreSQL::Test::BackgroundPsql)
 %global __provides_exclude ^perl\\((PostgresVersion|PostgresNode|RecursiveCopy|SimpleTee|TestLib|PostgreSQL::Test::BackgroundPsql)
 
-Summary:	PGSpider is High-Performance SQL Cluster Engine for distributed big data.
-Name:		%{sname_pgs}%{pgmajorversion}
-Version:	%{pgreleaseversion}
-Release:	%{?dist}
-License:	TOSHIBA CORPORATION
-Url:		https://github.com/pgspider
+Summary:	PostgreSQL client programs and libraries
+Name:		%{sname}%{pgmajorversion}
+Version:	16.0
+Release:	2PGDG%{?dist}
+License:	PostgreSQL
+Url:		https://www.postgresql.org/
 
-Source0:	pgspider-%{pgreleaseversion}.tar.bz2
+Source0:	https://download.postgresql.org/pub/source/v%{version}/postgresql-%{version}.tar.bz2
 Source4:	%{sname}-%{pgmajorversion}-Makefile.regress
 Source5:	%{sname}-%{pgmajorversion}-pg_config.h
 Source6:	%{sname}-%{pgmajorversion}-README-systemd.rpm-dist
 Source7:	%{sname}-%{pgmajorversion}-ecpg_config.h
 Source9:	%{sname}-%{pgmajorversion}-libs.conf
-Source12:	postgresql-16-A4.pdf
+Source12:	https://www.postgresql.org/files/documentation/pdf/%{pgpackageversion}/%{sname}-%{pgpackageversion}-A4.pdf
 %if 0%{?suse_version}
 Source14:	%{sname}-%{pgmajorversion}.pam.suse
 %else
@@ -248,28 +247,24 @@
 Requires(post):	%{_sbindir}/update-alternatives
 Requires(postun):	%{_sbindir}/update-alternatives
 
-Provides:	%{sname_pgs} >= %{version}-%{release}
+Provides:	%{sname} >= %{version}-%{release}
 
 %description
-PGSpider is High-Performance SQL Cluster Engine for distributed big data.
-PGSpider can access a number of data sources using Foreign Data Wrapper(FDW)
-and retrieves the distributed data source vertically. Usage of PGSpider is
-the same as PostgreSQL except its program name is pgspider and default port
-number is 4813. You can use any client applications such as libpq and psql.
-The base pgspider package contains the client programs that you'll need to
-access a PGSpider DBMS server, as well as HTML documentation for the whole
+PostgreSQL is an advanced Object-Relational database management system (DBMS).
+The base postgresql package contains the client programs that you'll need to
+access a PostgreSQL DBMS server, as well as HTML documentation for the whole
 system. These client programs can be located on the same machine as the
-PGSpider server, or on a remote machine that accesses a PGSpider server
-over a network connection. The PGSpider server can be found in the
-pgspider%{pgmajorversion}-server sub-package.
+PostgreSQL server, or on a remote machine that accesses a PostgreSQL server
+over a network connection. The PostgreSQL server can be found in the
+postgresql%{pgmajorversion}-server sub-package.
 
-If you want to manipulate a PGSpider database on a local or remote PGSpider
+If you want to manipulate a PostgreSQL database on a local or remote PostgreSQL
 server, you need this package. You also need to install this package
-if you're installing the pgspider%{pgmajorversion}-server package.
+if you're installing the postgresql%{pgmajorversion}-server package.
 
 %package libs
-Summary:	The shared libraries required for any PGSpider clients
-Provides:	pgspider-libs = %{pgmajorversion} libpq5 >= 10.0
+Summary:	The shared libraries required for any PostgreSQL clients
+Provides:	postgresql-libs = %{pgmajorversion} libpq5 >= 10.0
 
 %if 0%{?rhel} && 0%{?rhel} <= 6
 Requires:	openssl
@@ -292,13 +287,13 @@
 %endif
 
 %description libs
-The pgspider%{pgmajorversion}-libs package provides the essential shared libraries for any
-PGSpider client program or interface. You will need to install this package
-to use any other PGSpider package or any clients that need to connect to a
-PGSpider server.
+The postgresql%{pgmajorversion}-libs package provides the essential shared libraries for any
+PostgreSQL client program or interface. You will need to install this package
+to use any other PostgreSQL package or any clients that need to connect to a
+PostgreSQL server.
 
 %package server
-Summary:	The programs needed to create and run a PGSpider server
+Summary:	The programs needed to create and run a PostgreSQL server
 Requires:	%{name}%{?_isa} = %{version}-%{release}
 Requires:	%{name}-libs%{?_isa} = %{version}-%{release}
 Requires(pre):	/usr/sbin/useradd /usr/sbin/groupadd
@@ -318,7 +313,7 @@
 Requires(postun):	systemd
 %endif
 
-Provides:	pgspider-server >= %{version}-%{release}
+Provides:	postgresql-server >= %{version}-%{release}
 
 %if 0%{?rhel} && 0%{?rhel} == 7
 %ifarch ppc64 ppc64le
@@ -327,28 +322,28 @@
 %endif
 
 %description server
-PGSpider is an advanced Object-Relational database management system (DBMS).
-The pgspider%{pgmajorversion}-server package contains the programs needed to create
-and run a PGSpider server, which will in turn allow you to create
-and maintain PGSpider databases.
+PostgreSQL is an advanced Object-Relational database management system (DBMS).
+The postgresql%{pgmajorversion}-server package contains the programs needed to create
+and run a PostgreSQL server, which will in turn allow you to create
+and maintain PostgreSQL databases.
 
 %package docs
-Summary:	Extra documentation for PGSpider
-Provides:	pgspider-docs >= %{version}-%{release}
+Summary:	Extra documentation for PostgreSQL
+Provides:	postgresql-docs >= %{version}-%{release}
 
 %description docs
-The pgspider%{pgmajorversion}-docs package includes the SGML source for the documentation
+The postgresql%{pgmajorversion}-docs package includes the SGML source for the documentation
 as well as the documentation in PDF format and some extra documentation.
-Install this package if you want to help with the PGSpider documentation
+Install this package if you want to help with the PostgreSQL documentation
 project, or if you want to generate printed documentation. This package also
 includes HTML version of the documentation.
 
 %package contrib
-Summary:	Contributed source and binaries distributed with PGSpider
+Summary:	Contributed source and binaries distributed with PostgreSQL
 Requires:	%{name}%{?_isa} = %{version}-%{release}
 Requires:	%{name}-libs%{?_isa} = %{version}-%{release}
 Requires:	%{name}-server%{?_isa} = %{version}-%{release}
-Provides:	pgspider-contrib >= %{version}-%{release}
+Provides:	postgresql-contrib >= %{version}-%{release}
 
 %if 0%{?rhel} && 0%{?rhel} == 7
 %ifarch ppc64 ppc64le
@@ -357,11 +352,11 @@
 %endif
 
 %description contrib
-The pgspider%{pgmajorversion}-contrib package contains various extension modules that are
-included in the PGSpider distribution.
+The postgresql%{pgmajorversion}-contrib package contains various extension modules that are
+included in the PostgreSQL distribution.
 
 %package devel
-Summary:	PGSpider development header files and libraries
+Summary:	PostgreSQL development header files and libraries
 Requires:	%{name}%{?_isa} = %{version}-%{release}
 Requires:	%{name}-libs%{?_isa} = %{version}-%{release}
 %if %llvm
@@ -406,7 +401,7 @@
 %endif
 %endif
 
-Provides:	pgspider-devel >= %{version}-%{release}
+Provides:	postgresql-devel >= %{version}-%{release}
 Obsoletes:	libpq-devel <= 42.0
 
 %if 0%{?rhel} && 0%{?rhel} == 7
@@ -416,15 +411,15 @@
 %endif
 
 %description devel
-The pgspider%{pgmajorversion}-devel package contains the header files and libraries
+The postgresql%{pgmajorversion}-devel package contains the header files and libraries
 needed to compile C or C++ applications which will directly interact
-with a PGSpider database management server. It also contains the ecpg
+with a PostgreSQL database management server. It also contains the ecpg
 Embedded C Postgres preprocessor. You need to install this package if you want
-to develop applications which will interact with a PGSpider server.
+to develop applications which will interact with a PostgreSQL server.
 
 %if %llvm
 %package llvmjit
-Summary:	Just-in-time compilation support for PGSpider
+Summary:	Just-in-time compilation support for PostgreSQL
 Requires:	%{name}-server%{?_isa} = %{version}-%{release}
 %if 0%{?rhel} && 0%{?rhel} == 7
 %ifarch aarch64
@@ -443,7 +438,7 @@
 Requires:	llvm => 5.0
 %endif
 
-Provides:	pgspider-llvmjit >= %{version}-%{release}
+Provides:	postgresql-llvmjit >= %{version}-%{release}
 
 %if 0%{?rhel} && 0%{?rhel} == 7
 %ifarch ppc64 ppc64le
@@ -452,21 +447,21 @@
 %endif
 
 %description llvmjit
-The pgspider%{pgmajorversion}-llvmjit package contains support for
-just-in-time compiling parts of PGSpider queries. Using LLVM it
+The postgresql%{pgmajorversion}-llvmjit package contains support for
+just-in-time compiling parts of PostgreSQL queries. Using LLVM it
 compiles e.g. expressions and tuple deforming into native code, with the
 goal of accelerating analytics queries.
 %endif
 
 %if %plperl
 %package plperl
-Summary:	The Perl procedural language for PGSpider
+Summary:	The Perl procedural language for PostgreSQL
 Requires:	%{name}-server%{?_isa} = %{version}-%{release}
 Requires:	perl(:MODULE_COMPAT_%(eval "`%{__perl} -V:version`"; echo $version))
 %ifarch ppc ppc64
 BuildRequires:	perl-devel
 %endif
-Provides:	pgspider-plperl >= %{version}-%{release}
+Provides:	postgresql-plperl >= %{version}-%{release}
 
 %if 0%{?rhel} && 0%{?rhel} == 7
 %ifarch ppc64 ppc64le
@@ -475,18 +470,18 @@
 %endif
 
 %description plperl
-The pgspider%{pgmajorversion}-plperl package contains the PL/Perl procedural language,
-which is an extension to the PGSpider database server.
+The postgresql%{pgmajorversion}-plperl package contains the PL/Perl procedural language,
+which is an extension to the PostgreSQL database server.
 Install this if you want to write database functions in Perl.
 
 %endif
 
 %if %plpython3
 %package plpython3
-Summary:	The Python3 procedural language for PGSpider
+Summary:	The Python3 procedural language for PostgreSQL
 Requires:	%{name}%{?_isa} = %{version}-%{release}
 Requires:	%{name}-server%{?_isa} = %{version}-%{release}
-Provides:	pgspider-plpython3 >= %{version}-%{release}
+Provides:	postgresql-plpython3 >= %{version}-%{release}
 %if 0%{?suse_version} >= 1315
 Requires:	python3-base
 %else
@@ -501,18 +496,18 @@
 %endif
 
 %description plpython3
-The pgspider%{pgmajorversion}-plpython3 package contains the PL/Python3 procedural language,
-which is an extension to the PGSpider database server.
+The postgresql%{pgmajorversion}-plpython3 package contains the PL/Python3 procedural language,
+which is an extension to the PostgreSQL database server.
 Install this if you want to write database functions in Python 3.
 %endif
 
 %if %pltcl
 %package pltcl
-Summary:	The Tcl procedural language for PGSpider
+Summary:	The Tcl procedural language for PostgreSQL
 Requires:	%{name}%{?_isa} = %{version}-%{release}
 Requires:	%{name}-server%{?_isa} = %{version}-%{release}
 Requires:	tcl
-Provides:	pgspider-pltcl >= %{version}-%{release}
+Provides:	postgresql-pltcl >= %{version}-%{release}
 
 %if 0%{?rhel} && 0%{?rhel} == 7
 %ifarch ppc64 ppc64le
@@ -521,17 +516,17 @@
 %endif
 
 %description pltcl
-PGSpider is an advanced Object-Relational database management
+PostgreSQL is an advanced Object-Relational database management
 system. The %{name}-pltcl package contains the PL/Tcl language
 for the backend.
 %endif
 
 %if %test
 %package test
-Summary:	The test suite distributed with PGSpider
+Summary:	The test suite distributed with PostgreSQL
 Requires:	%{name}-server%{?_isa} = %{version}-%{release}
 Requires:	%{name}-devel%{?_isa} = %{version}-%{release}
-Provides:	pgspider-test >= %{version}-%{release}
+Provides:	postgresql-test >= %{version}-%{release}
 
 %if 0%{?rhel} && 0%{?rhel} == 7
 %ifarch ppc64 ppc64le
@@ -540,13 +535,13 @@
 %endif
 
 %description test
-The pgspider%{pgmajorversion}-test package contains files needed for various tests for the
-PGSpider database management system, including regression tests and
+The postgresql%{pgmajorversion}-test package contains files needed for various tests for the
+PostgreSQL database management system, including regression tests and
 benchmarks.
 %endif
 
 %prep
-%setup -q -n %{sname_pgs}-%{version}
+%setup -q -n %{sname}-%{version}
 
 %patch -P 1 -p0
 %patch -P 3 -p0
@@ -560,7 +555,7 @@
 # fail quickly and obviously if user tries to build as root
 %if %runselftest
 	if [ x"`id -u`" = x0 ]; then
-		echo "pgspider's regression tests fail if run as root."
+		echo "postgresql's regression tests fail if run as root."
 		echo "If you really need to build the RPM as root, use"
 		echo "--define='runselftest 0' to skip the regression tests."
 		exit 1
@@ -676,7 +671,7 @@
 # run_testsuite WHERE
 # -------------------
 # Run 'make check' in WHERE path. When that command fails, return the logs
-# given by PGSpider build system and set 'test_failure=1'.
+# given by PostgreSQL build system and set 'test_failure=1'.
 
 run_testsuite()
 {
@@ -783,10 +778,10 @@
 # backups of data go here...
 %{__install} -d -m 700 %{buildroot}/var/lib/pgsql/%{pgmajorversion}/backups
 
-# Create the multiple PGSpider version startup directory
+# Create the multiple PostgreSQL version startup directory
 %{__install} -d -m 700 %{buildroot}/etc/sysconfig/pgsql/%{pgmajorversion}
 
-# Install linker conf file under pgspider installation directory.
+# Install linker conf file under postgresql installation directory.
 # We will install the latest version via alternatives.
 %{__install} -d -m 755 %{buildroot}%{pgbaseinstdir}/share/
 %{__install} -m 700 %{SOURCE9} %{buildroot}%{pgbaseinstdir}/share/
@@ -823,10 +818,6 @@
 # Fix some more documentation
 # gzip doc/internals.ps
 %{__cp} %{SOURCE6} README.rpm-dist
-cd doc/src/sgml
-%{__make} html
-%{__make} man
-cd ../../..
 %{__mkdir} -p %{buildroot}%{pgbaseinstdir}/share/doc/html
 %{__mv} doc/src/sgml/html doc
 %{__mkdir} -p %{buildroot}%{pgbaseinstdir}/share/man/
@@ -895,7 +886,7 @@
 %pre server
 groupadd -g 26 -o -r postgres >/dev/null 2>&1 || :
 useradd -M -g postgres -o -r -d /var/lib/pgsql -s /bin/bash \
-	-c "PGSpider Server" -u 26 postgres >/dev/null 2>&1 || :
+	-c "PostgreSQL Server" -u 26 postgres >/dev/null 2>&1 || :
 
 %post server
 /sbin/ldconfig
@@ -1117,9 +1108,6 @@
 %{pgbaseinstdir}/lib/pg_visibility.so
 %{pgbaseinstdir}/lib/pg_walinspect.so
 %{pgbaseinstdir}/lib/postgres_fdw.so
-%{pgbaseinstdir}/lib/pgspider_fdw.so
-%{pgbaseinstdir}/lib/pgspider_core_fdw.so
-%{pgbaseinstdir}/lib/pgspider_keepalive.so
 %{pgbaseinstdir}/lib/refint.so
 %{pgbaseinstdir}/lib/seg.so
 %if %ssl
@@ -1182,8 +1170,6 @@
 %{pgbaseinstdir}/share/extension/pgrowlocks*
 %{pgbaseinstdir}/share/extension/pgstattuple*
 %{pgbaseinstdir}/share/extension/postgres_fdw*
-%{pgbaseinstdir}/share/extension/pgspider_fdw*
-%{pgbaseinstdir}/share/extension/pgspider_core_fdw*
 %{pgbaseinstdir}/share/extension/refint*
 %{pgbaseinstdir}/share/extension/seg*
 %if %ssl
@@ -1240,7 +1226,7 @@
 %{pgbaseinstdir}/bin/pg_test_timing
 %{pgbaseinstdir}/bin/pg_upgrade
 %{pgbaseinstdir}/bin/pg_verifybackup
-%{pgbaseinstdir}/bin/pgspider
+%{pgbaseinstdir}/bin/postgres
 %{pgbaseinstdir}/share/man/man1/initdb.*
 %{pgbaseinstdir}/share/man/man1/pg_archivecleanup.1
 %{pgbaseinstdir}/share/man/man1/pg_checksums.*
@@ -1306,7 +1292,6 @@
 %{pgbaseinstdir}/lib/libpgtypes.a
 %{pgbaseinstdir}/lib/pgxs/*
 %{pgbaseinstdir}/lib/pkgconfig/*
-%{pgbaseinstdir}/lib/pgspider_keepalive.so
 %{pgbaseinstdir}/share/man/man1/ecpg.*
 
 %if %llvm
@@ -1348,27 +1333,42 @@
 %endif
 
 %changelog
-* Tue Jul 11 2023 t-kataym - 3.1.0
-- Features:
-    - Support migration data command
-    - Support routing insert
-    - Parallel stored function
-    - Support modify table bulk insert
-- Bug fixes:
-    - Fix crash nil pointer
-    - Fix Inconsistent intervalstyle in multi threads
-    - Fix crash when pgspider child node has no spdurl
-    - Fix crash on tupdesc because of race conditions
-    - Fix crash when changing transaction state
-    - Fix returning slot in wrong memory context
-    - Fix do not call EndForeignScan if child thread make error
-    - Fix bug cannot find child datasources - There is no candidate to INSERT
-    - Free guc variable
-    - Remove wrong condition check when copying string variable
+* Tue Sep 19 2023 Devrim Gunduz <devrim@gunduz.org> - 16.0-2PGDG
+- Re-add plpython3 (was plpython) build macro for the users who don't
+  want to build with PL/Python (make it consistent with other PLs)
 
-* Tue Jan 17 2023 t-kataym - 3.0.0
-- RELEASE v3.0.0
+* Mon Sep 11 2023 Devrim Gunduz <devrim@gunduz.org> - 16.0-1PGDG
+- Update to v16.0 Gold!
 
-* Fri Dec 24 2021 t-kataym - 2.0.0
-- RELEASE v2.0.0
+* Tue Aug 29 2023 Devrim Gunduz <devrim@gunduz.org> - 16.0-rc1-1PGDG
+- Update to v16 RC 1
+- Remove LLVM dependency in the main package on SLES 15. Per bug report
+  from Muralikrishna Bandaru.
+
+* Tue Aug 8 2023 Devrim Gunduz <devrim@gunduz.org> - 16.0-beta3-1
+- Update to v16 beta3
+
+* Thu Jun 29 2023 Devrim Gunduz <devrim@gunduz.org> - 16.0-beta2-1
+- Update to v16 beta2
+
+* Sun Jun 4 2023 Devrim Gündüz <devrim@gunduz.org> - 16.0-beta1-2
+- Rebuild against LLVM 15 on SLES 15
+
+* Tue May 23 2023 Devrim Gunduz <devrim@gunduz.org> - 16.0-beta1-1
+- Update to v16 beta1
+- Remove a few configure parameters: plpython3 and systemd_enabled are no longer
+  needed, as they are supported on all distros. Also remove %%disablepgfts macro.
+
+* Mon Apr 24 2023 Devrim Gunduz <devrim@gunduz.org> - 16-alpha_20230424_PGDG.1
+- Modernise %patch usage, which has been deprecated in Fedora 38
+
+* Tue Feb 7 2023 Devrim Gündüz <devrim@gunduz.org> - 16-alpha_20230208_PGDG.1
+- Enable TAP tests on all RHEL versions, per report from Bill Smith.
+  Commit 4f08cb8d41b8 is now obsolete, it seems.
+
+* Mon Dec 05 2022 Devrim Gündüz <devrim@gunduz.org> - 16-alpha_20221205_PGDG.1
+- Get rid of AT and switch to GCC on RHEL 7 - ppc64le
+
+* Mon Aug 8 2022 Devrim Gündüz <devrim@gunduz.org> - 16.0-alpha
+- Initial cut for PostgreSQL 16.
 
